clash-vexriscv/src/VexRiscv/JtagTcpBridgeHaskell.hs

```diff
@@ -0,0 +1,186 @@
+-- SPDX-FileCopyrightText: 2023 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+module VexRiscv.JtagTcpBridge where
+
```

Comment:
We should probably remove this file, it's not used anywhere at the moment and I'm not super happy with the design and the code, so I'm happy to have it gone probably :D